src/ycc_hull/controllers/notifications/helpers_notifications_controller.py

```diff
@@ -158,20 +369,22 @@ async def _send_overdue_tasks_reminder(
             EmailMessageBuilder()
             .to(contact)
             .reply_to(contact)
-            .subject(n_overdue_tasks_str)
+            .subject(f"⛵⏰ {n_overdue_tasks_str}")
             .content(
                 f"""
-<p>Dear {contact.first_name},</p>
+<p>Dear {contact.first_name} {_BOAT_PARTY},</p>
 
-<p>This is a reminder that you are the contact for {n_overdue_tasks_str}.</p>
+<p>This is a reminder that you are the contact for {n_overdue_tasks_str}. ⏰</p>
 
 {format_helper_tasks_list(tasks)}
 
 <p>You can:</p>
 
 <ul>
     <li>Validate the tasks. During validation you will be asked to mark which members showed up and optionally you can leave a comment.</li>
-    <li>If the task was not done before the deadline, maybe you want to extend it.</li>
+    <li>If the task was not done before the deadline, maybe you want to extend it to allow more time for helpers to do it.</li>
+    <li>If you wish to cancel a task, you can do it by validating it (feel free to comment that it is cancelled).</li>
+    <li>If you wish to delete a task, please cancel it instead.</li>
```

Comment:
a bit confusing to read. maybe just better to say above:

"If you wish to delete or cancel a task, .."

Reply:
Thank you - this email only goes to boat maintainers and there is a constant  confusion between cancelling or deleting a task (deleting is not good for YCC, as we should award a shifts to members who signed up before someone wants to cancel it, so cancellation is basically validation)